dhall-lsp-server/src/Dhall/LSP/Backend/Parsing.hs

```diff
@@ -77,9 +90,7 @@ getLetAnnot (Src left _ text) = Megaparsec.parseMaybe (unParser parseLetAnnot) t
 --   containing `x`. Returns the original Src if something goes wrong.
 getLetIdentifier :: Src -> Src
 getLetIdentifier src@(Src left _ text) =
-  case Megaparsec.parseMaybe (unParser parseLetIdentifier) text of
-    Just src' -> src'
-    Nothing -> src
+  Maybe.fromMaybe src $ Megaparsec.parseMaybe (unParser parseLetIdentifier) text
```

Comment:
I used to like `fromMaybe` but I've since realized that I find the explicit pattern matching more readable.

Could we ignore this lint?